rest-api/dao/organization_hierarchy_sync_dao.py

```diff
@@ -0,0 +1,385 @@
+import os
+import logging
+import googlemaps
+import lib_fhir.fhirclient_3_0_0.models.organization
+
+from lib_fhir.fhirclient_3_0_0.models.fhirabstractbase import FHIRValidationError
+from werkzeug.exceptions import BadRequest
+from dao.base_dao import BaseDao
+from model.hpo import HPO
+from model.organization import Organization
+from model.site import Site
+from model.site_enums import ObsoleteStatus, SiteStatus, EnrollingStatus, DigitalSchedulingStatus
+from dao.hpo_dao import HPODao
+from participant_enums import OrganizationType
+from dao.organization_dao import OrganizationDao
+from dao.site_dao import SiteDao
+from dateutil.parser import parse
+
+_FHIR_SYSTEM_PREFIX = 'http://all-of-us.org/fhir/sites/'
```

Comment:
We have a similar structure in api_util.py. We might consider consolidating this constant there.

Reply:
done